splunk_topology/check.py

```diff
@@ -125,7 +125,9 @@ def check(self, instance):
         except Exception as e:
             self._clear_topology(instance_key, clear_in_snapshot=True)
             self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.CRITICAL, tags=instance.tags, message=str(e))
-            raise CheckException("Splunk topology failed with message: %s" % e), None, sys.exc_info()[2]
+            if not instance.splunk_ignore_saved_search_errors:
+                raise CheckException("Splunk topology failed with message: %s" % e), None, sys.exc_info()[2]
+            self.log.warning("Splunk topology exception occured and ignored as ignore_saved_search_errors flag is true.")
```

Comment:
maybe log the actual error message when the flag is set to true.

Reply:
maybe include a piece of specific information in where the error occurred? This message is very similar as to next message. This message may be fired on dispatching while the other might be fired during finalization.